IntelBluetoothFirmware.xcodeproj/project.pbxproj

```diff
@@ -444,20 +422,24 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ARCHS = x86_64;
+				CLANG_WARN_DOCUMENTATION_COMMENTS = NO;
 				CODE_SIGN_STYLE = Manual;
```

Comment:
Same here. Replace `CODE_SIGN_STYLE = Manual;` with `CODE_SIGN_IDENTITY = "";` might be a good idea.

Reply:
Same as above.